diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -216,6 +218,14 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    private fun checkForComplexExpression(node: ASTNode) {
+        if (node.getRootNode().getFilePath().isGradleScript()) {
```

Comment:
Where this method `isGradle` comes from? Is it our extension? 
May be one day we can move this logic to Rule class, so it will skip some rules for some groups of patterns.

Reply:
Yes, it's from one of our utils files.